src/graph/sampling/neighbor/neighbor.cc

```diff
@@ -90,6 +91,8 @@ HeteroSubgraph SampleNeighbors(
         default:
           LOG(FATAL) << "Unsupported sparse format.";
       }
+      if (!exclude_edges.empty() && exclude_edges[etype].GetSize() != 0)
+        sampled_coo = COORemove(sampled_coo, exclude_edges[etype]);
```

Comment:
what is the complexity of using this function? `SampleNeighbors` runs on the CSC format. adding this function requires to generate the COO format?